src/main/java/org/opentripplanner/transit/model/organization/Agency.java

```diff
@@ -26,9 +27,17 @@ public final class Agency extends AbstractTransitEntity<Agency, AgencyBuilder> i
 
   Agency(AgencyBuilder builder) {
     super(builder.getId());
+    // Fill agency name, if not exists take short name
+    String nameValue = (builder.getName() != null && !builder.getName().isBlank())
```

Comment:
This code should like in the mapper so that we don't need to change the internal data model.